mod_tokens.md

```diff
@@ -12,9 +12,9 @@ With this cached information they know to which eMSP they can later send a CDR.
 
 ### 1.1 Push model
 
-When the MSP creates a new Token object they push it to the CPO by calling [PUT](#212-put-method) on the CPOs Tokens endpoint with the newly created Token object.
+When the MSP creates a new Token object they push it to the CPO by calling [PUT](#212-put-method) on the CPO‘s Tokens endpoint with the newly created Token object.
```

Comment:
`‘`  is a single turned comma quotation mark (U+2018)
What we want here is an apostrophe: `'` (U+0027)

Reply:
the rest of the changes is fine. please push a corrected version.
And thanks for taking the time to improve the specs!